test/e2e/multiple_policies_for_same_node.go

```diff
@@ -0,0 +1,98 @@
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+func ifaceUpWithStaticIP(iface string, ipAddress string) nmstatev1alpha1.State {
```

Comment:
Could you make all these helpers into local functions of the `Context` they are needed in? We have quite a lot of these and they are shared across all the tests. Either make them private

```
ifaceUpWithStaticIP = func(...) ...
```

Or move them to `test/e2e/states.go`